qunit-dom/package.json

```diff
@@ -14,16 +13,31 @@
   "repository": "https://github.com/simplabs/qunit-dom",
   "license": "MIT",
   "author": "simplabs GmbH",
-  "types": "dist/qunit-dom.d.ts",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/addon-test-support/qunit-dom-modules.d.ts",
+      "default": "./dist/addon-test-support/index.js"
+    }
   },
+  "typesVersions": {
+    "*": {
+      "*": "dist/addon-test-support/qunit-dom-modules.d.ts"
+    }
+  },
+  "types": "dist/qunit-dom.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "build": "rollup -c",
+    "build": "rollup -c ./rollup.config.mjs",
     "changelog": "lerna-changelog",
-    "lint": "eslint . --cache",
-    "lint:fix": "eslint . --fix",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
+    "lint:package": "publint",
```

Comment:
I set up publint and are-the-types-wrong, because I couldn't debug my exports/typesVersions, turns out I had a typo :sweat_smile: 

good tools that are allowed once `type=module` is present in the package.json